### ISSUE TITLE
Notify admins when space is low.

### ISSUE THREAD

**Issue by zenhack** [sandstorm-io/sandstorm#3341]
I think we've discussed this a bit before, but it came up on IRC again today and I don't think we have an open issue for it so:

Sandstorm should warn users about low disk space, preferably *before* it is too low for sandstorm to succesfully start.

**Comment by ocdtrekkie** [comment]
Yeah, I discussed this in #3204 but we focused that one on user's storage usage.
